.gitignore

```diff
@@ -7,3 +7,4 @@ yarn-error.log*
 *.swp
 *.swo
 tmp
+pnpm-lock.yaml
```

Comment:
Please remove the changes in `.gitignore` as they are not related to the concept entry.